### PR TITLE
ts-web/core: docs additions

### PR DESCRIPTION
**Pull request by pro-wh**
suggested in #1160

### CODE REVIEW

File: client-sdk/ts-web/core/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # An oasis-core SDK for TypeScript
 
-Developers, see [this getting started guide](docs/getting-started.md).
+Users of this package, see [this getting started guide](docs/getting-started.md).
```
Comment (pro-wh) [inline client-sdk/ts-web/core/README.md:3]: we already had a link to a sample envoy configuration. clarifying this link to indicate that users of this package should see the guide there

File: client-sdk/ts-web/core/docs/getting-started.md [hunk 2]
```diff
@@ -33,6 +36,11 @@ For security and performance reasons, some methods are not permitted.
 See the Advanced deployment section below for how to set up your own node with
 a custom set of enabled methods.
 
+## Running in Node.js
+
+This package and some of its dependencies do not support running in Node.js.
+Only use this package in browsers.
```
[the remark this answers is not in the document]
Comment (kostko) [reply client-sdk/ts-web/core/docs/getting-started.md:42]: Has it been tested to work? If yes, we could downgrade, otherwise just leave as-is.

**Review by kostko — APPROVED** [review]

File: client-sdk/ts-web/core/docs/getting-started.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (pro-wh) [reply client-sdk/ts-web/core/docs/getting-started.md:42]: no. only that there are some unit tests we run in node

**Comment by pro-wh** [comment]
wait what

**Comment by pro-wh** [comment]
ugh I'll have to reopen

**Pull request merged**